app/views/comments/_comment_header.html.erb

```diff
@@ -0,0 +1,34 @@
+<header class="comment__header">
+  <a href="/<%= comment.user.username %>" class="crayons-link crayons-link--secondary flex items-center fw-medium">
```

Comment:
I recommend using our `URL` helper module as much as possible instead of manually building links:

```suggestion
  <a href="<%= URL.user(comment.user) %>" class="crayons-link crayons-link--secondary flex items-center fw-medium">
```

Reply:
Oh nice, I'll replace it wherever I can! :)